FordLabs/retroquest
# Thought Dialog Redesign

**WARNING, LOOK AT PR #73 BEFORE THIS ONE! THAT BRANCH WAS MERGED INTO THIS ONE.**

## Overview
The thought dialogs were still using the old design, which does not fit in with the current styleguide. The effort was done to use the new dialogs for thoughts as defined in the style guide.

### Demo
<img width="1122" alt="screen shot 2018-08-02 at 9 29 27 am" src="https://user-images.githubusercontent.com/6293337/43586806-99bf07a2-9636-11e8-965f-40d7a1ad0ba1.png">
<img width="1073" alt="screen shot 2018-08-02 at 9 29 33 am" src="https://user-images.githubusercontent.com/6293337/43586810-9d3dde12-9636-11e8-826c-ee3dfd370a4b.png">

## Testing Instructions
1. Add a thought to any of the three columns and click on the message part of the thought. 
2. You should see a dialog popup that looks like the screenshots provided. 
3. Try out all of the CRUD operations and verify that they all work correctly.
4. The functionality of the dialog should behave exactly like the thoughts via CRUD operations.
